index.html

```diff
@@ -238,17 +238,17 @@ <h3>Locales and Internationalization</h3>
         
         <p class="definition"><dfn data-lt="common locale data repository|CLDR">Common Locale Data Repository</dfn> (or <em>[[CLDR]]</em>). The Common Locale Data Repository is a Unicode Consortium project that defines, collects, and curates sets of data needed to enable <a>locales</a> in systems or operating environments. CLDR data and its locale model are widely adopted, particularly in browsers.</p>
         
-        <p class="definition"><dfn data-lt="unicode locale|unicode locale identifier|unicode locale identifiers|unicode locales">Unicode Locale Identifier</dfn> or <em>Unicode Locale</em>. A <a>language tag</a> that follows the additional processing rules defined by [[CLDR]] in UTR#35 [[LDML]]. A Unicode Locale can include a combination of certain language tag extensions ([[RFC6067]], [[RFC6497]]), although it is not required to do so.</p>
-        
-        <p>A Unicode locale provides the ability to specify in a <a>language tag</a> some <a>international preference</a> variations that go beyond linguistic or regional variation or to select formatting behavior or content when there are multiple options or user preferences within a given locale.  Unicode locale identifiers are well-formed [[BCP47]] language tags. [[CLDR]] also specifies some additional rules about the structure and content of the Unicode Locale's language tag as well as supplying specific interpretation of certain subtags. See <a href="https://www.unicode.org/reports/tr35/tr35.html#Unicode_locale_identifier">Section 3.2</a> of [[LDML]] for details. </p>
+        <p class="definition"><dfn data-lt="unicode locale|unicode locale identifier|unicode locale identifiers|unicode locales">Unicode Locale Identifier</dfn> or <em>Unicode Locale</em>. A <a>language tag</a> that follows the additional rules and restrictions on subtag choice defined in UTR#35 [[LDML]]. Any valid Unicode locale identifier is also a <a>valid</a> [[BCP47]] <a>language tag</a>, but a few <a>valid</a> <a>language tags</a> are not also valid Unicode locale identifiers.</p>
+                
+        <p>[[CLDR]] defines and maintains two language tag extensions ([[RFC6067]] and [[RFC6497]]) that are related to <a>Unicode locale identifiers</a>. These extensions allow a <a>language tag</a> to express some <a>international preference</a> variations that go beyond linguistic or regional variation or to select formatting behavior or content when there are multiple options or user preferences within a given locale. <a>Unicode locale identifiers</a> are not required to include these extensions: they are only used when the locale being identified requires additional tailoring provided by one of these extensions. [[CLDR]] also applies specific interpretation of certain subtags when used as a locale identifier. See <a href="https://www.unicode.org/reports/tr35/tr35.html#Unicode_locale_identifier">Section 3.2</a> of [[LDML]] for details.</p>
```

Comment:
at least as far as the extension, the rfcs are also related to BCP47 language tags, but perhaps we could say 'which are particularly relevant to Unicode locale identifiers, and define the sequence of tags that follow a -u or -t extension" ?

Reply:
You haven't explained the meaning of "but a few valid language tags are not also valid Unicode locale identifiers".  Perhaps a single sentence para could be include here to say something like "Unicode locale identifiers come with a few rules about choice of subtags in a language tag: for example, deprecated subtags should be replaced with non-deprecated alternatives."

Reply:
By the way, that "but a few valid language tags are not also valid Unicode locale identifiers" is very helpful.  Nice addition!

Reply:
The RFCs are not just "related to BCP47 language tag". They *are* valid parts of language tags--part of BCP47--via the extension mechanism.

Your second comment is very helpful. See if these changes help.

Reply:
LGTM